Makefile

```diff
@@ -4,6 +4,8 @@ ifndef TAG
 	TAG=dev-alpine
 endif
 
+PORT ?= 8080
```

Comment:
I would use a less generic envvar to avoid collision. Maybe something like ```FRSS_PORT```.

Reply:
The command is usually used as `PORT=8000 make start` (i.e. no need to export) so I don't mind having a generic variable name. If we change to `FRSS_PORT`, we should change `TAG` as well to be coherent

Reply:
That makes sense

Reply:
Or `make start PORT=23423` (doesn't change the point). :-)